src/applications/personalization/profiles-wrapper/ProfilesWrapper.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { Suspense } from 'react';
+import { connect } from 'react-redux';
+import { BrowserRouter, Route, Switch, Redirect } from 'react-router-dom';
+import FEATURE_FLAG_NAMES from 'platform/utilities/feature-toggles/featureFlagNames';
+import Profile2Wrapper from 'applications/personalization/profile-2/components/Profile2Wrapper';
+import ProfileOneWrapper from 'applications/personalization/profile360/containers/VAProfileApp';
+import environment from 'platform/utilities/environment';
+import routes from 'applications/personalization/profile-2/routes';
+import { isLOA3 as isLOA3Selector } from 'platform/user/selectors';
+import { selectShowProfile2 } from 'applications/personalization/profile-2/selectors';
+
+const LoadingPage = <Profile2Wrapper>Loading...</Profile2Wrapper>;
+
+const ProfilesWrapper = ({ showProfile2, isLOA3 }) => {
+  if (!showProfile2) {
+    return <ProfileOneWrapper />;
+  }
+
+  return (
+    <BrowserRouter>
+      <Suspense fallback={LoadingPage}>
+        <Switch>
+          {routes.map(route => {
+            if (route.requiresLOA3 && !isLOA3) {
```

Comment:
When the user hits a route that they do not have access to we redirect them to /personal-information.

Reply:
Doing a redirect seems fine but won't we want to send them to the account security section since that will be the only section they have access to if they are LOA1?

Reply:
Yeah, I think we talked about a ticket yesterday that implied that behavior. I will follow up with Matt to make sure that's what we want. In this ticket I just wanted to set up the basics (to make this redirecting easily configurable) as it seemed out of scope, but it would be an easy change 👍